client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -113,14 +113,16 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 	const generatedDesigns = allDesigns?.generated?.designs || [];
 	const staticDesigns = allDesigns?.static?.designs || [];
 
+	const hasVerticalImages = generatedDesigns.length > 0;
```

Comment:
It's a little weird that using the number of the generated designs indicates whether we have vertical images or not. For example, if the user is in the “Sell” flow, they won't see the generated designs but we still have vertical images, and those standard themes will be verticalized as well.

Do we also need to avoid verticalizing standard themes in the “Sell” flow?

Reply:
I guess we could get the value from the current selected vertical as the vertical endpoint has already returned this field. What do you think?

Reply:
You mean we call the vertical endpoint in the UDP step? As we only store the site vertical ID from the vertical step 🥲

Reply:
How about storing the selected vertical in the `ONBOARD_STORE`?

Reply:
I have another radical idea:

- Assume that `signup/standard-theme-v13n` is always true
- Move the standard theme whitelist to wpcom
- Return `is_verticalized` field for each design from the starter-designs endpoint. The value is = (the design is verticalizable) AND (the vertical has images)

This way, Calypso does not need to check whether the current vertical has images. On Calypso side, the code is cleaner.

I've pushed the changes for Calypso in this PR, and for wpcom side here: D89293-code

I've updated the testing instructions as well, to include the Seller flow

Let me know what you think!

Reply:
> Do we also need to avoid verticalizing standard themes in the “Sell” flow?

I suppose if that is how it has been, then leave it as is.    We could flag this in #dotcom-create.  Honestly I didn't ever think about this, but it seems ok to me.  I mean users in this flow still see the v13n screen, so it doesn't seem like it is a problem to show the 4 standard-v13n in any flow.  If it shows in build & write, then ok for sell I think. 

Also, let's keep some perspective. This is one small tracks event.  It shouldn't occupy days of dev time.   I don't know enough to understand why it is so different to harness `has_vertical_images` here since other events/props have this.   I guess it is because the sell flow operates differently from write/build.  

But if this is so complicated in the code, then maybe it is smarter to solve the issue on the data analysis side.  e.g. setting up funnels a special way or doing a HUE query.  Meaning it might be more difficult to do the analysis but that is ok.

Reply:
If you have it solved though, then cool!!!!   🥳

Reply:
> This is one small tracks event. It shouldn't occupy days of dev time. I don't know enough to understand why it is so different to harness `has_vertical_images` here since other events/props have this.

The problem is that "`has_vertical_images` that other events/props have" turns out to be wrong, and the fix is not trivial. But yeah we've solved it, and aiming to finish it in a few hours!